kolibri/plugins/base.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+"""The base of a Kolibri plugin is the inheritence from
+:class:`.KolibriPluginBase`.
+
+.. moduleauthor:: Learning Equality <info@learningequality.org>
+
+"""
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import absolute_import
+
+
+class MandatoryPluginMethodNotImplemented(Exception):
+
+    def __init__(self):
+        NotImplemented.__init__(self, "Plugin needs to define this method")
```

Comment:
Oh wait sorry, yes, that's a typo.. you can't inherit from NotImplemented, and this one is a left over, well spotted! :)